### PR TITLE
Includes LabelBot Cross Account Log Functionality

### LINKED ISSUE #21: [feature request] have cloudwatch logs of the label bot under the CI team account viewable/readable externally
It would be nice to see the logs from cloudwatch for the label bot. Purpose of this is so that if a user faces an issue with the label bot not performing as expected - we can look at logs and determine what is going on.

### PR DESCRIPTION
**Pull request by harshp8l**
Logs are now viewable in the ai-mxnet-eng account for the label bot
Fixes #21

### CODE REVIEW

**Comment by harshp8l** [comment]
Updated log access PR to remove the need of account id in README (assuming user should know how to switch roles given description in README)

File: mxnet-bot/LabelBotFullFunctionality/README.md [hunk 1]
```diff
@@ -77,3 +77,14 @@ As well, make sure to set the appropriate CNAME certificate from Certificate Man
...
 ***Note:*** Confirm in each update that the basePath is set to /dev and point it to corret lambda under Custom Domain Names in API Gateway.
+
+
+#### 5. CloudWatch Log Access
+* In (deploy.sh) fill in the variables 
+    - dest_account="AWS_ACCOUNT_NUM" with the AWS account of the destination of the logs
+    - bot_account="AWS_ACCOUNT_NUM" with the AWS account of the bot account
+* Then, the destination log account proceeds to assume the role (to have view access of the logs)
+* After assuming the role proceed to view the logs by navigating to the appropriate section in the CloudWatch console
+   - [Label Logs](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logStream:group=/aws/lambda/LabelBot-dev-label)
```
Comment (lanking520) [inline mxnet-bot/LabelBotFullFunctionality/README.md]: Who has the permission to access this link?

File: mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh [hunk 2]
```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 # Script to deploy the label bot to our dev account
 set -e
-
 export AWS_PROFILE=mxnet-ci-dev
+
+dest_account="106780517423"
+bot_account="968277166688"
+
+sed "s/###HERE###/$dest_account/g" log-access-trust-policy.json-template > log-access-trust-policy.json
```
Comment (lanking520) [inline mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh]: Nice hack!

Comment (harshp8l) [reply mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh]: Actually I will update this to leave those account spots blank

File: mxnet-bot/LabelBotFullFunctionality/README.md (hunk 1 above)
Comment (harshp8l) [reply mxnet-bot/LabelBotFullFunctionality/README.md]: Only those who are given access to view the label bot logs (or else its a dead link to everyone else)

File: mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh (hunk 2 above)
Comment (larroy) [reply mxnet-bot/LabelBotFullFunctionality/deploy_bot.sh]: I would use python formatting, but ok.

**Pull request merged**